src/install.ts

```diff
@@ -58,18 +73,23 @@ export default async function install(): Promise<void> {
     const isTemplateIncluded: boolean = answers.modules.includes('template');
 
     const configTemplate: string = await fs.readFile(
-        path.resolve(__dirname, '../templates/config.template'),
+        path.resolve(__dirname, '../templates/config.ejs'),
         'utf-8'
     );
-
-    let config: string = configTemplate
-        .replace('<steps>', JSON.stringify([...stepsPackages].map(p => 'node_modules/' + p)))
-        .replace('<format>', JSON.stringify(
+    const configEjs = ejs.compile(configTemplate);
+    const config = configEjs({
+        steps: JSON.stringify([...stepsPackages].map(p => 'node_modules/' + p)),
```

Comment:
maybe it is better to replace 'node_modules/' + p to `node_modules/${p}`

Reply:
fixed